digit-health-services/health-sync-service/src/main/java/org/digit/health/sync/repository/SyncQueryBuilder.java

```diff
@@ -0,0 +1,34 @@
+package org.digit.health.sync.repository;
+
+import lombok.extern.slf4j.Slf4j;
+import org.digit.health.sync.web.models.request.SyncSearchDto;
+import org.springframework.stereotype.Component;
+
+@Slf4j
+@Component
+public class SyncQueryBuilder {
```

Comment:
Make an interface SyncLogQueryBuilder and implement with DefaultSyncLogQueryBuilder